2023/03/24 11:36:26 [error] [php] Undefined variable $sandip (/Users/sandipp/Sites/localhost/yiitest/protected/views/user/create.php:15)
Stack trace:
#0 /Users/sandipp/Sites/localhost/yiitest/framework/web/CBaseController.php(126): require()
#1 /Users/sandipp/Sites/localhost/yiitest/framework/web/CBaseController.php(95): UserController->renderInternal()
#2 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(872): UserController->renderFile()
#3 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(785): UserController->renderPartial()
#4 /Users/sandipp/Sites/localhost/yiitest/protected/controllers/UserController.php(78): UserController->render()
#5 /Users/sandipp/Sites/localhost/yiitest/framework/web/actions/CInlineAction.php(49): UserController->actionCreate()
#6 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(308): CInlineAction->runWithParams()
#7 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilterChain.php(134): UserController->runAction()
#8 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilter.php(40): CFilterChain->run()
#9 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(1148): CAccessControlFilter->filter()
#10 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CInlineFilter.php(58): UserController->filterAccessControl()
#11 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter()
#12 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(291): CFilterChain->run()
#13 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(265): UserController->runActionWithFilters()
#14 /Users/sandipp/Sites/localhost/yiitest/framework/web/CWebApplication.php(282): UserController->run()
#15 /Users/sandipp/Sites/localhost/yiitest/framework/web/CWebApplication.php(141): CWebApplication->runController()
#16 /Users/sandipp/Sites/localhost/yiitest/framework/base/CApplication.php(185): CWebApplication->processRequest()
#17 /Users/sandipp/Sites/localhost/yiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yiitest/index.php/user/create
in /Users/sandipp/Sites/localhost/yiitest/protected/views/user/create.php (15)
in /Users/sandipp/Sites/localhost/yiitest/protected/controllers/UserController.php (78)
in /Users/sandipp/Sites/localhost/yiitest/index.php (13)
2023/03/24 11:54:54 [error] [php] include(User.php): Failed to open stream: No such file or directory (/Users/sandipp/Sites/localhost/yiitest/framework/YiiBase.php:463)
Stack trace:
#0 /Users/sandipp/Sites/localhost/yiitest/framework/YiiBase.php(463): include()
#1 /Users/sandipp/Sites/localhost/yiitest/protected/controllers/UserController.php(65): autoload()
#2 /Users/sandipp/Sites/localhost/yiitest/framework/web/actions/CInlineAction.php(49): UserController->actionCreate()
#3 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(308): CInlineAction->runWithParams()
#4 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilterChain.php(134): UserController->runAction()
#5 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilter.php(40): CFilterChain->run()
#6 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(1148): CAccessControlFilter->filter()
#7 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CInlineFilter.php(58): UserController->filterAccessControl()
#8 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter()
#9 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(291): CFilterChain->run()
#10 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(265): UserController->runActionWithFilters()
#11 /Users/sandipp/Sites/localhost/yiitest/framework/web/CWebApplication.php(282): UserController->run()
#12 /Users/sandipp/Sites/localhost/yiitest/framework/web/CWebApplication.php(141): CWebApplication->runController()
#13 /Users/sandipp/Sites/localhost/yiitest/framework/base/CApplication.php(185): CWebApplication->processRequest()
#14 /Users/sandipp/Sites/localhost/yiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yiitest/index.php/user/create
in /Users/sandipp/Sites/localhost/yiitest/protected/controllers/UserController.php (65)
in /Users/sandipp/Sites/localhost/yiitest/index.php (13)
2023/03/24 11:57:13 [error] [php] include(User.php): Failed to open stream: No such file or directory (/Users/sandipp/Sites/localhost/yiitest/framework/YiiBase.php:463)
Stack trace:
#0 /Users/sandipp/Sites/localhost/yiitest/framework/YiiBase.php(463): include()
#1 /Users/sandipp/Sites/localhost/yiitest/protected/controllers/UserController.php(65): autoload()
#2 /Users/sandipp/Sites/localhost/yiitest/framework/web/actions/CInlineAction.php(49): UserController->actionCreate()
#3 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(308): CInlineAction->runWithParams()
#4 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilterChain.php(134): UserController->runAction()
#5 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilter.php(40): CFilterChain->run()
#6 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(1148): CAccessControlFilter->filter()
#7 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CInlineFilter.php(58): UserController->filterAccessControl()
#8 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter()
#9 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(291): CFilterChain->run()
#10 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(265): UserController->runActionWithFilters()
#11 /Users/sandipp/Sites/localhost/yiitest/framework/web/CWebApplication.php(282): UserController->run()
#12 /Users/sandipp/Sites/localhost/yiitest/framework/web/CWebApplication.php(141): CWebApplication->runController()
#13 /Users/sandipp/Sites/localhost/yiitest/framework/base/CApplication.php(185): CWebApplication->processRequest()
#14 /Users/sandipp/Sites/localhost/yiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yiitest/index.php/user/create
in /Users/sandipp/Sites/localhost/yiitest/protected/controllers/UserController.php (65)
in /Users/sandipp/Sites/localhost/yiitest/index.php (13)
2023/03/24 12:01:21 [error] [exception.CException] CException: User has an invalid validation rule. The rule must specify attributes to be validated and the validator name. in /Users/sandipp/Sites/localhost/yiitest/framework/base/CModel.php:289
Stack trace:
#0 /Users/sandipp/Sites/localhost/yiitest/framework/base/CModel.php(260): CModel->createValidators()
#1 /Users/sandipp/Sites/localhost/yiitest/framework/base/CModel.php(304): CModel->getValidators('username')
#2 /Users/sandipp/Sites/localhost/yiitest/framework/web/helpers/CHtml.php(1534): CModel->isAttributeRequired('username')
#3 /Users/sandipp/Sites/localhost/yiitest/framework/web/widgets/CActiveForm.php(601): CHtml::activeLabelEx(Object(User), 'username', Array)
#4 /Users/sandipp/Sites/localhost/yiitest/protected/views/user/_form.php(26): CActiveForm->labelEx(Object(User), 'username')
#5 /Users/sandipp/Sites/localhost/yiitest/framework/web/CBaseController.php(126): require('/Users/sandipp/...')
#6 /Users/sandipp/Sites/localhost/yiitest/framework/web/CBaseController.php(95): CBaseController->renderInternal('/Users/sandipp/...', Array, true)
#7 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(872): CBaseController->renderFile('/Users/sandipp/...', Array, true)
#8 /Users/sandipp/Sites/localhost/yiitest/protected/views/user/create.php(21): CController->renderPartial('_form', Array)
#9 /Users/sandipp/Sites/localhost/yiitest/framework/web/CBaseController.php(126): require('/Users/sandipp/...')
#10 /Users/sandipp/Sites/localhost/yiitest/framework/web/CBaseController.php(95): CBaseController->renderInternal('/Users/sandipp/...', Array, true)
#11 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(872): CBaseController->renderFile('/Users/sandipp/...', Array, true)
#12 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(785): CController->renderPartial('create', Array, true)
#13 /Users/sandipp/Sites/localhost/yiitest/protected/controllers/UserController.php(78): CController->render('create', Array)
#14 /Users/sandipp/Sites/localhost/yiitest/framework/web/actions/CInlineAction.php(49): UserController->actionCreate()
#15 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#16 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#17 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilter.php(40): CFilterChain->run()
#18 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(1148): CFilter->filter(Object(CFilterChain))
#19 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#20 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#21 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(291): CFilterChain->run()
#22 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 /Users/sandipp/Sites/localhost/yiitest/framework/web/CWebApplication.php(282): CController->run('create')
#24 /Users/sandipp/Sites/localhost/yiitest/framework/web/CWebApplication.php(141): CWebApplication->runController('user/create')
#25 /Users/sandipp/Sites/localhost/yiitest/framework/base/CApplication.php(185): CWebApplication->processRequest()
#26 /Users/sandipp/Sites/localhost/yiitest/index.php(13): CApplication->run()
#27 {main}
REQUEST_URI=/yiitest/index.php/user/create
HTTP_REFERER=http://localhost:8888/yiitest/index.php/site/login
---
2023/03/24 12:11:08 [error] [exception.ParseError] ParseError: syntax error, unexpected token "public", expecting end of file in /Users/sandipp/Sites/localhost/yiitest/protected/models/User.php:49
Stack trace:
#0 /Users/sandipp/Sites/localhost/yiitest/protected/controllers/UserController.php(69): YiiBase::autoload('User')
#1 /Users/sandipp/Sites/localhost/yiitest/framework/web/actions/CInlineAction.php(49): UserController->actionCreate()
#2 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#3 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#4 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilter.php(40): CFilterChain->run()
#5 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(1148): CFilter->filter(Object(CFilterChain))
#6 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 /Users/sandipp/Sites/localhost/yiitest/framework/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#8 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(291): CFilterChain->run()
#9 /Users/sandipp/Sites/localhost/yiitest/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /Users/sandipp/Sites/localhost/yiitest/framework/web/CWebApplication.php(282): CController->run('create')
#11 /Users/sandipp/Sites/localhost/yiitest/framework/web/CWebApplication.php(141): CWebApplication->runController('user/create')
#12 /Users/sandipp/Sites/localhost/yiitest/framework/base/CApplication.php(185): CWebApplication->processRequest()
#13 /Users/sandipp/Sites/localhost/yiitest/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yiitest/index.php/user/create
HTTP_REFERER=http://localhost:8888/yiitest/index.php/user/create
---
2024/01/10 05:14:15 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://localhost/
---
2024/01/10 06:57:26 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://localhost/
---
2024/01/10 07:07:52 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://demo.darwinboxlocal.com/
---
2024/01/10 08:19:29 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://demo.darwinboxlocal.com/
---
2024/01/10 09:02:03 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://demo.darwinboxlocal.com/
---
2024/01/10 09:22:32 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://demo.darwinboxlocal.com/
---
2024/01/10 10:52:06 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://demo.darwinboxlocal.com/
---
2024/01/10 11:08:51 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://demo.darwinboxlocal.com/index.php
---
2024/01/10 11:10:19 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://localhost/
---
2024/01/10 12:38:08 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://demo.darwinboxlocal.com/
---
2024/01/12 14:02:33 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://localhost/
---
2024/01/17 12:42:15 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://demo.darwinboxlocal.com/
---
2024/02/29 11:37:19 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in /data/live/framework/web/CWebApplication.php:286
Stack trace:
#0 /data/live/framework/web/CWebApplication.php(141): CWebApplication->runController()
#1 /data/live/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /data/live/index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://localhost/
---
